simtools/configuration/commandline_parser.py

```diff
@@ -231,26 +236,218 @@ def initialize_simulation_model_arguments(self, model_options):
+            List of simulation software configuration parameters.
+        """
+        if simulation_configuration is None:
+            return
+
+        if "software" in simulation_configuration:
+            self._initialize_simulation_software()
+        if "corsika_configuration" in simulation_configuration:
+            self._initialize_simulation_configuration()
+            self._initialize_shower_configuration()
+
+    def _initialize_simulation_software(self):
+        """Initialize simulation software arguments."""
+        _software_group = self.add_argument_group("simulation software")
+        _software_group.add_argument(
+            "--simulation_software",
+            help="Simulation software steps.",
+            type=str,
+            choices=["corsika", "simtel", "corsika_simtel"],
+            required=True,
+            default="corsika_simtel",
+        )
+
+    def _initialize_simulation_configuration(self):
+        """Initialize simulation configuration arguments."""
+        _configuration_group = self.add_argument_group("simulation configuration")
+        _configuration_group.add_argument(
+            "--primary",
+            help="Primary particle to simulate.",
+            type=str.lower,
+            required=True,
+            choices=[
+                "gamma",
+                "gamma_diffuse",
+                "electron",
+                "proton",
+                "muon",
+                "helium",
+                "nitrogen",
+                "silicon",
+                "iron",
+            ],
+        )
+        _configuration_group.add_argument(
+            "--azimuth_angle",
+            help=(
+                "Telescope pointing direction in azimuth. "
+                "It can be in degrees between 0 and 360 or one of north, south, east or west "
+                "North is 0 degrees and the azimuth grows clockwise (East is 90 degrees)."
+            ),
+            type=CommandLineParser.azimuth_angle,
+            required=True,
+        )
+        _configuration_group.add_argument(
+            "--zenith_angle",
+            help="Zenith angle in degrees (between 0 and 180).",
+            type=CommandLineParser.zenith_angle,
+            required=True,
+        )
+        _configuration_group.add_argument(
+            "--nshow",
+            help="Number of showers per run to simulate.",
+            type=int,
+            required=False,
+        )
+        _configuration_group.add_argument(
+            "--run_number_start",
+            help="Run number for the first run.",
+            type=int,
+            required=True,
+            default=1,
+        )
+        _configuration_group.add_argument(
+            "--number_of_runs",
+            help="Number of runs to be simulated.",
+            type=int,
+            required=True,
+            default=1,
+        )
+        _configuration_group.add_argument(
+            "--event_number_first_shower",
```

Comment:
first shower event number?

Reply:
I prefer it this way, as the important point is the event number.